Behavioral.Automation.Bindings.UI.Tests/Configurations/Hooks.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Behavioral.Automation.Bindings.UI.Abstractions;
+using BoDi;
+using TechTalk.SpecFlow;
+
+namespace Behavioral.Automation.Bindings.UI.Tests.Configurations;
+
+[Binding]
+public class Configuration
+{
+    private readonly IObjectContainer _objectContainer;
+
+    public Configuration(IObjectContainer objectContainer)
+    {
+        _objectContainer = objectContainer;
+    }
+
+    [BeforeScenario()]
```

Comment:
Redundant parentheses if you don't want to use `Scenario` filters

Reply:
Removed